specs-go/config.go

```diff
@@ -112,12 +112,11 @@ type Platform struct {
 
 // Mount specifies a mount for a container.
 type Mount struct {
-	// Destination is the path where the mount will be placed relative to the container's root.  The path and child directories MUST exist, a runtime MUST NOT create directories automatically to a mount point.
+	// Destination is the absolute path where the mount will be placed in the container.
 	Destination string `json:"destination"`
 	// Type specifies the mount kind.
-	Type string `json:"type,omitempty"`
-	// Source specifies the source path of the mount.  In the case of bind mounts on
-	// Linux based systems this would be the file on the host.
+	Type string `json:"type,omitempty" platform:"linux,solaris"`
```

Comment:
> No use of the word `additional`.

In the `process.user` case, the platform-specific sections cover the *only* properties for that object.  In this case, the Linux/Solaris section will be adding a new property to the existing platform-independent properties.  The best precedent we have for that is probably [here][1], which doesn't use “additional” either.  I'd be happier with “additional” or some other language that makes it clear that both the platform-independent properties and the POSIX-specific `type` are valid entries, but I'm ok punting that clarification to follow-up work.

I'd rather have the other bits changed too (e.g. we do use the hypenated “Linux-based” [here][1]), but I'm fine punting those to follow-up work as well.

[1]: https://github.com/opencontainers/runtime-spec/blame/2ec18c02474dd49ce20aa30236959a8a80763042/config.md#L158